src/platform/site-wide/cta-widget/helpers.js

```diff
@@ -252,6 +259,9 @@ export const serviceDescription = appId => {
     case widgetTypes.DIRECT_DEPOSIT:
       return 'change your direct deposit information online';
 
+    case widgetTypes.DISABILITY_RATINGS:
+      return 'view your VA disability rating';
+
```

Comment:
Anything I'm missing here? Seems suspiciously simple.

Reply:
Well, if this isn't all that's required it'll at least get you part of the way there!